src/app/middleware/handleErrors.ts

```
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from 'express';
+import { ValidationError } from '@hapi/joi';
+
+export default (error: Error, _request: Request, response: Response, _next: NextFunction) => {
+    switch (error.name) {
+        case 'ValidationError':
+            return handleValidationError(error as ValidationError, response);
+        default:
+            return response.status(500).send('An unexpected error has occured.');
+    }
+}
```

Nice ol' catch all handler 👌